site/app/templates/grading/electronic/ta_status/StatusManualGradingHistogram.twig

```diff
@@ -0,0 +1,101 @@
+{# Graph Setup for Manual Grading #}
```

Comment:
This twig seems to be mostly identical to `StatusAutogradingHistogram.twig`, what about using ternary operator and re-use existing `StatusAutogradingHistogram.twig` to achieve the same result.

Reply:
I see the value in doing so, but having different files for each type of histogram seems the most consistent with the way things have been previously organized. Since before adding StatusManualGradingHistogram.twig, there was the StatusOverallHistogram.twig file that is also almost identical to StatusAutogradingHistogram.twig. 

If we want to update it to reduce redundancy, we can have one file to include generation for all three of the histograms. In that case, I can use a ternary operator to do so. What are your thoughts?

Reply:
That is a good point, I think this is a good time to do refactor, it shouldn't be difficult to merge it into one twig.